package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "splunk-logging",
-  "version": "0.11.1",
+  "version": "0.11.2000",
```

Comment:
I chose a version number that likely won't be used by the splunk team so we can easily identify if our package is being used